pkg/operator/resources/syncapi/k8s_specs.go

```diff
@@ -637,10 +637,6 @@ func getEnvVars(api *spec.API, container string) []kcore.EnvVar {
 				Name:  "CORTEX_THREADS_PER_PROCESS",
 				Value: s.Int32(api.Predictor.ThreadsPerProcess),
 			},
-			kcore.EnvVar{
-				Name:  "CORTEX_MAX_REPLICA_CONCURRENCY",
```

Comment:
Thanks for catching this merge error. On second thought `CORTEX_MAX_REPLICA_CONCURRENCY` is the more accurate term name. Perhaps we should rename `CORTEX_MAX_PROCESS_CONCURRENCY` to `CORTEX_MAX_REPLICA_CONCURRENCY`

Reply:
By the look of it, `CORTEX_MAX_PROCESS_CONCURRENCY` is computed by taking the `max_replica_concurrency` (the old term being `CORTEX_MAX_REPLICA_CONCURRENCY`) and dividing it by the number of processes `processes_per_replica`. So, it doesn't look like it's the same thing. The current variable that holds `CORTEX_MAX_REPLICA_CONCURRENCY` seems to be `CORTEX_MAX_CONN`.